lib/resources/port.rb

```diff
@@ -8,7 +8,8 @@
 # Usage:
```

Comment:
Since we are at it, lets remove the usage comment, since everything is covered within the `example` block